.circleci/config.yml

```diff
@@ -152,6 +152,65 @@ jobs:
           name: Build
           command: yarn run build
 
+  integration_tests:
+    docker:
+      - image: circleci/python:3.7.3-node-browsers
+    
+        environment:
+          REACT_APP_INSECURE_AUTH_ALLOWED: true
+          MOZ_HEADLESS: 1
+
+    steps:
+      - checkout
+
+      - setup_remote_docker
+
+      - restore_cache:
+          keys:
+          - v2-dependencies-{{ checksum "yarn.lock" }}
+          # fallback to using the latest cache if no exact match is found
+          - v2-dependencies-
+          - integration-test-dependencies-{{ checksum "tests/integration/requirements.txt" }}
+
+      - run:
+          name: Install node dependencies
+          command: yarn install
+
+      - save_cache:
+          paths:
+            - "node_modules"
+          key: v2-dependencies-{{ checksum "yarn.lock" }}
+
+      - run:
+          name: Install Tox
+          command: sudo pip install tox
+      
+      - run:
+          name: Firefox Version Check
+          command: firefox --version
+
+      - run:
```

Comment:
this entire step should be omitted and `.therapist.yml` should be updated to include a `black` action:
```yaml
black:
    run: black --check --diff --line-length 99 {files}
    fix: black --line-length 99 {files}
    include: "*.py"
```

Note: The `88` line length seemed arbitrary so I updated to 99 to be consistent with Normandy.

Reply:
Should I just specify the ```tests/integration``` folder?

Reply:
i don't think that's necessary and if we ever add any other python files we likely want it to apply to those as well.